### PR TITLE
Add `avatar` property

### PR DESCRIPTION
**Pull request by Charliekenney23**
Hi,

Thanks for the package. I'm Working on an NPM client right now and I needed to get the user's avatar so I figured I'd add it. I also updated the tests so that they would pass (had to update your user info and use es6 arrow CBs).

- Charles

### CODE REVIEW

**Comment by Charliekenney23** [comment]
Failing because test library requires 'const' but node 0.x doesn't support it.

**Comment by sindresorhus** [comment]
Don't do unrelated changes. ES5 => ES6 should be a separate PR if so.

File: test.js [hunk 1]
```diff
@@ -1,13 +1,24 @@
 import test from 'ava';
 import fn from './';
 
-test(async t => {
+test('should scrape ~sindresorhus', async t => {
 	const user = await fn('sindresorhus');
 
-	t.is(user.name, 'Sindre Sorhus');
+	t.is(user.name, null);
```
Comment (sindresorhus) [inline test.js]: You can't just set this to `null`. If it returns `null` it's clearly something wrong going on in the existing code.

**Comment by Charliekenney23** [comment]
@sindresorhus Yeah I realize that now. Do you wanna just close this and I'll open up two PRs? Let me know whether I should disable xo or if you're going to change the travis config.

File: test.js (hunk 1 above)
Comment (0xch4z) [reply test.js]: Your name really is null.
- [yours](https://www.npmjs.com/~sindresorhus)
- [mine (with name)](https://www.npmjs.com/~charleskenney)

**Comment by sindresorhus** [comment]
Nah, it's fine. We can do this one. Just remember focused PRs next time. :)

File: test.js (hunk 1 above)
Comment (sindresorhus) [reply test.js]: Wut. That's super weird. Seems like npm had a bug which nulled all my info... I remember having all that set up at least half a year ago.

Comment (sindresorhus) [reply test.js]: I've added back the info now on npm, so you can revert these changes.

**Comment by Charliekenney23** [comment]
Sorry about that, still learning about PR/Commit conventions.

**Comment by sindresorhus** [comment]
No worries :)

**Comment by sindresorhus** [comment]
Thanks 👍

**Comment by sindresorhus** [comment]
New release is out now.

**Pull request merged**